reporting/csv-reporter.js

```diff
@@ -51,7 +60,7 @@ CsvReporter.prototype.write = function(timestamp, name, header, line, values) {
       // include header if file didn't previously exist
       data = util.format("t,%s\n%s", header, data);
     }
-    fs.appendFile(file, data);
+    fs.appendFile(file, data, {}, noop);
```

Comment:
Probably the right way to fix this would be to make this callback based, and then allow `report()` to optionally take a callback (or return a promise).  This way users can detect completion of `report()`.  The main use case is for that is to prevent simultaneous `report` instances.   In the case of `CsvReporter` its probably not a problem because you likely won't be reporting as frequently as it takes to append to a file.  There are a few other enhancements that could be made to `ScheduledReporter` that i'll open up an issue for.